### PR TITLE
First version of new york times clone

### PR DESCRIPTION
**Pull request by TommyM0ntana**

### CODE REVIEW

**Comment by tomzacchia** [comment]
- W3 HTML validation (Note, we did not add any empty elements to get  rid of warnings since they would be redundant) 
Screen Shot 2019-09-24 at 11 50 57 AM

- The theory of inflation infographic. We designed this part for 1440x604 resolution
Screen Shot 2019-09-24 at 11 52 25 AM

**Review by Forison — COMMENTED** [review]
# pull request under review by Addo

**Review by Forison — CHANGES_REQUESTED** [review]
Hi Team, 👋👋
Your project looks nice, am impress, but there are a few additions to make.
- #### **[REQUIRED]** README 🙂 
Your readme file is supposed to be descriptive enough(readme has to describe the project you have worked on in detail. It can include information on collaborators, the technology used, why the project was designed, how to access, etc... ).[readme](https://github.com/Forison/Using-Bootstrap)

- #### **[REQUIRED]** PROJECT IMPLEMENTATION 🙂
your project does not display all the contents found on the original. Your project ends with the theory of inflation.
* There is no footer
>note here the footer is the fixed bottom navbar like element at the bottom of your screen with inscription "access more of the times by creating account
* I also noticed that you used inline styling in a couple of places in your code. Inline styling is a bad practice you should shy away from using them. all styling must be done in your ```main.css```  per this project.
###  VALIDATOR WARNING ISSUES
There are two ways to avoid this warning:
 * Either you do what you suggested by adding  ```<h2> -<h6>```, which would be of course obsolete😔 or
* Replace those ```section```  tags with ```div``` (I  recommend this approach)🙂 

> please don't forget to comment your pull request don't leave it empty
# Kindly make those changes and submit for review again
Happy coding

File: index.html [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+          Astrophysical Journal.
+        </p>
+      </div>
+    </section>
+
+    <aside class="side-content">
+      <!-- Sign up -->
+      <div class="sign-up">
+        <p> Sign up for Science Times </p>
+        <p style="font-size: 12px">
```
Comment (Forison) [inline index.html]: > inline styling, not good practice

File: index.html [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+  <!-- Infaltion infographic -->
+  <section class="artwork">
+    <h1> The Theory of Inflation </h1>
+    <p class="artwork_text">
+      Astronomers have found evidence to support the theory of inflation, 
+      which explains how the universe expanded so uniformly and 
+      so quickly in the instant after the Big Bang 13.8 billion years ago. 
+    </p>
+
+    <div class="theories" style="background-image: url('./images/inflation.png')">
```
Comment (Forison) [inline index.html]: > inline styling, not good practice

File: index.html [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+      <div class="theory-text-container"> 
+        <p>
+          <span style="font-weight: bold"> THE UNIVERSE </span>
```
Comment (Forison) [inline index.html]: > inline styling, not good practice

File: index.html [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+        </p>
+        <p>
+          <span style="font-weight: bold"> NOT ENOUGH TIME </span> 
+          The universe is not old enough for light to have traveled 
+          the vast distance from one side of the universe to the other, 
+          and there has not been enough time for scattered patches of hot 
+          and cold to mix into an even temperature.
+        </p>
+        <p>
+          <span style="font-weight: bold"> THE UNIVERSE </span>
```
Comment (Forison) [inline index.html]: > rectify all other inline stylings as mentioned in the main review

**Review by Forison — CHANGES_REQUESTED** [review]

**Comment by tomzacchia** [comment]
- updated readme
- added footer and floating sign in navbar thingy
- removed inline styling
- for the w3 warnings, we left the sections as they were since it is a requirement to use semantic HTML

**Review by uche-inyama — CHANGES_REQUESTED** [review]
#### Hi Tommy, :wave: 

I can see you have implemented some of the changes suggested by my colleague. However, you missed some. 

- [ ] Your html document is not valid, there still errors showing up, when passed through  [html validator](https://validator.w3.org).

- [ ] There are inline styling in your html document, all css implementation should be done in your main.css.

#### Implement these changes, then submit another form for code review.

**Comment by tomzacchia** [comment]
Screen Shot 2019-09-26 at 11 31 30 AM

**Review by uche-inyama — APPROVED** [review]
Hi Tommy, we are good now :slightly_smiling_face: 

#### This is project is approved
**```Go ahead and merge to master branch```** :clap: :clap:

**Pull request merged**